README.md

```diff
@@ -39,8 +39,7 @@ Everytime the image is updated, all tags will be pushed with the latest updates.
 
 ## Pre-Requisites
 
-- install docker-compose [https://docs.docker.com/compose/install/](https://docs.docker.com/compose/install/)
```

Comment:
The link should be a markdown link. It just so happens that github is rendering the url as a link, other markdown viewers may not.